kubernetes-templates/kube-dns/0/README.md

```diff
@@ -0,0 +1,6 @@
+# KubeDNS
+
+Kubernetes offers a DNS cluster addon, which most of the supported environments
```

Comment:
we shouldn't mention cluster add on here. Our services won't be enabled using the add on manager.